core/tests/test_utils.py

```diff
@@ -286,15 +286,23 @@ def check_image_png_or_webp(image_string: str) -> bool:
 
 
 def get_storage_model_module_names() -> Iterator[models.Names]:
-    """Get all module names in storage."""
+    """Get all module names in storage.
+
+    Yields:
+        Iterator. The Iterator object.
```

Comment:
Please take a look at how iterator values are defined in other files.
example:
https://github.com/oppia/oppia/blob/8d821797c8b941347bf0b33d110d0923da96209e/core/jobs/jobs_manager.py#L257

Reply:
I have updated the docstring.